src/components/DownloadTrajectoryMenu/index.tsx

```diff
@@ -54,7 +54,8 @@ const DownloadTrajectoryMenu = ({
     };
 
     const isDisabled = !fileIsLoaded() || isBuffering;
-    const tooltipOffset = isDisabled ? [25, -30] : [40, -18];
+    // disabled buttons are wrapped in a span which changes the tooltip offset
+    const tooltipOffset = isDisabled ? [-30, -20] : [0, -8];
```

Comment:
the fix was to lower the tooltip

Reply:
Since these numbers show up in two places, is there a common place to store them that is maybe related to the offset created by the span you reference in the comment?